wolfcrypt/src/chacha.c

```diff
@@ -408,12 +420,10 @@ static void chacha_encrypt_avx(ChaCha* ctx, const byte* m, byte* c,
     byte*  output;
     word32 i;
     word32 cnt = 0;
-    static const word64 add[2] =  { 0x0000000100000000UL,0x0000000300000002UL };
-    static const word64 four[2] = { 0x0000000400000004UL,0x0000000400000004UL };
-    static const word64 rotl8[2] =
-                                  { 0x0605040702010003UL,0x0e0d0c0f0a09080bUL };
-    static const word64 rotl16[2] =
-                                  { 0x0504070601000302UL,0x0d0c0f0e09080b0aUL };
```

Comment:
Had to go back to __m128i.
The previous change was to remove __m256i and I thought i could get rid off __m128i as well.
Old compilers didn't like my change so I returned to __m128i.